mmv1/third_party/tgc/ancestrymanager/ancestrymanager_test.go

```diff
@@ -500,6 +511,36 @@ func TestGetAncestors(t *testing.T) {
 			want:       []string{"organizations/unknown"},
 			wantParent: "//cloudresourcemanager.googleapis.com/organizations/unknown",
 		},
+		{
+			name: "Org-level CuOP set with parent field",
+			data: tfdata.NewFakeResourceData(
+				"google_org_policy_custom_constraint",
+				p.ResourcesMap["google_org_policy_custom_constraint"].Schema,
+				map[string]interface{}{
+					"parent": "organizations/12345",
+				},
+			),
+			asset: &resources.Asset{
+				Type: "orgpolicy.googleapis.com/CustomConstraint",
+			},
+			want:       []string{"organizations/12345"},
+			wantParent: "//cloudresourcemanager.googleapis.com/organizations/12345",
+		},
+		{
+			name: "Folder-level CuOP set with parent field",
+			data: tfdata.NewFakeResourceData(
+				"google_org_policy_custom_constraint",
```

Comment:
`google_org_policy_custom_constraint` is not an good resource to test `Folder-level` CuOP set with parent field, as its parent can only be `organization`. Are there any other resources to replace it?

Reply:
`google_folder` is a good candidate, and there's already test cases for that. In fact, this change allows us to remove the switch cases for the folder and custom IAM role resources. PTAL.